python/cudf_polars/cudf_polars/experimental/dask_serialize.py

```diff
@@ -0,0 +1,59 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: Apache-2.0
+
+"""Dask serialization."""
+
+from __future__ import annotations
+
+from distributed.protocol import dask_deserialize, dask_serialize
+from distributed.protocol.cuda import cuda_deserialize, cuda_serialize
+from distributed.utils import log_errors
+
+import pylibcudf as plc
+import rmm
+
+from cudf_polars.containers import DataFrame
+
+__all__ = ["register"]
+
+
+def register() -> None:
```

Comment:
You may have already discussed this point. But why not keep these registration functions at the module level instead of inside `def register`? That way they're registered on import.